### DISCUSSION TITLE
I want to know if exchangelib python librarie work also with microsoft exchange 2016-2019

### DISCUSSION THREAD

**Discussion by YasmineeBa** [ecederstrand/exchangelib#1266] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
If this is concerns missing functionality from EWS, please provide links to this feature in MSDN.

**Comment by ecederstrand** [comment]
Yes, it does. Currently, the following versions are known to work: https://github.com/ecederstrand/exchangelib/blob/7bf720d439bebf65f5e9f2ff2900b5fa9aa6c400/exchangelib/version.py#L117
